src/screens/DomainScreen/elements/Header/elements/CredderInfoGroup.js

```diff
@@ -73,14 +73,14 @@ const styles = StyleSheet.create({
     },
     tooltipShadowContainer: {
         paddingHorizontal: 10,
-        shadowColor: COLORS.black,
+        shadowColor: '#000',
         shadowOffset: {
             width: 0,
             height: 12,
         },
         shadowOpacity: 0.58,
         shadowRadius: 16,
-        shadowColor: COLORS.black,
+        shadowColor: 'black',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There is a redundancy in the `shadowColor` property. It's defined twice with the same value 'black'. You should remove one of them.
```diff
     tooltipShadowContainer: {
         paddingHorizontal: 10,
-        shadowColor: '#000',
         shadowOffset: {
             width: 0,
             height: 12,
         },
         shadowOpacity: 0.58,
         shadowRadius: 16,
         shadowColor: 'black',
         elevation: 16,
```
Also, there are commented out lines for `shadowRadius`. If these are not needed, consider removing them to keep the code clean.


<!-- This is an auto-generated comment by OSS CodeRabbit -->